Detections/VectraAI/VectraDetect-Host-by-Severity.yaml

```diff
@@ -0,0 +1,65 @@
+id: 8c705e06-6aa4-436a-b92f-0fa3fb248db6
+name: Vectra AI Detect - Suspected Compromised Host
+description: |
+  'Create an incident when a Host is suspected to be compromised. 
+  The higher the severity level is, the more immediate attention it requires as Vectra AI engine is more confident that this is a real threat. 
+  Level of severity are: Low, Medium, High, Critical). Recommended configuration is to trigger an alert for at least High and Critical.'
+severity: High
+requiredDataConnectors:
+  - connectorId: AIVectraDetect
+    dataTypes:
+      - CommonSecurityLog
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - CredentialAccess
+  - Discovery
+  - LateralMovement
+  - Collection
+  - CommandAndControl
+  - Exfiltration
+  - Impact
+relevantTechniques:
+query: |
+  // Edit this variable to only keep the Severity level where an incident needs to be created (Defaults are: "Low", "Medium", "High", "Critical" )   
+  let configured_level = dynamic(["Low", "Medium", "High", "Critical"]);
+  CommonSecurityLog
+  | where DeviceVendor == "Vectra Networks"
+  | where DeviceProduct == "X Series"
+  | where DeviceEventClassID == "hsc"
+  | project-rename threat_score = FlexNumber1
+  | project-rename certainty_score = FlexNumber2
+  | project-rename vectra_URL = DeviceCustomString4
+  | project-rename detection_name = DeviceEventClassID
+  | project-rename score_decreases = DeviceCustomString3
+  | extend level = case( threat_score <  50 and certainty_score < 50, "Low",
+                         threat_score < 50 and certainty_score >= 50 , "Medium", 
+                         threat_score >= 50 and certainty_score <= 50, "High", 
+                         threat_score >= 50 and certainty_score >= 50, "Critical",
+                         "UNKNOWN")
+  | extend Severity = case( level == "Low", "Low",
+                            level == "Medium", "Medium",
+                            level == "High", "Medium",
+                            level == "Critical", "High",
+                            "UNKNOWN")
+  | where level in (configured_level) 
+  //keep only the event with the highest threat score per Host
+  | summarize arg_max(threat_score, *) by SourceHostName
+  | project SourceHostName, level, Severity, TimeGenerated, SourceIP, threat_score, certainty_score, vectra_URL
```

Comment:
Do we want to map the different entity in the end using extend.
https://github.com/Azure/Azure-Sentinel/wiki/Query-Style-Guide#old-entity-mapping-method

Reply:
added